### PR TITLE
media-sound/beets and related: unmask

### PR DESCRIPTION
**Pull request by a17r**
Please note that media-sound/beets was not tested for missing python3_6 support on my current system. The updated dependencies are the only change there though.

### CODE REVIEW

File: dev-python/audioread/audioread-2.1.8.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+inherit distutils-r1
+
+DESCRIPTION="Wrapper for audio file decoding using FFmpeg or GStreamer"
+HOMEPAGE="https://pypi.org/project/audioread/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
```
Comment (SoapGentoo) [inline dev-python/audioread/audioread-2.1.8.ebuild]: `RESTRICT` clause?

File: dev-python/audioread/audioread-2.1.8.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+BDEPEND="
+	test? (
+		dev-python/pytest[${PYTHON_USEDEP}]
+		dev-python/pytest-runner[${PYTHON_USEDEP}]
+	)
+"
+RDEPEND="
+	|| ( (
```
Comment (SoapGentoo) [inline dev-python/audioread/audioread-2.1.8.ebuild]: the formatting here is off

File: dev-python/audioread/files/audioread-2.1.8-test-deps.patch [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+@@ -36,11 +36,8 @@ def _read(fn):
+ 
+       packages=['audioread'],
+ 
+-      setup_requires=[
+-          'pytest-runner'
+-      ],
+-
+       tests_require=[
++          'pytest-runner',
```
Comment (SoapGentoo) [inline dev-python/audioread/files/audioread-2.1.8-test-deps.patch]: I would delete this line too

File: dev-python/python-mpd/python-mpd-1.0.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+DEPEND="twisted? ( dev-python/twisted[${PYTHON_USEDEP}] )"
+RDEPEND="${DEPEND}"
+
+DOCS=( doc/changes.rst doc/topics/{advanced,commands,getting-started,logging}.rst README.rst )
+
+S="${WORKDIR}/${PN}2-${PV}"
+
+distutils_enable_tests setup.py
+
+src_prepare() {
```
Comment (SoapGentoo) [inline dev-python/python-mpd/python-mpd-1.0.0.ebuild]: `python_prepare_all`

File: media-sound/beets/beets-1.4.9-r1.ebuild [hunk 5]
```diff
@@ -78,6 +78,8 @@ RDEPEND="${DEPEND}"
 
 RESTRICT="test" # tests broken in 1.4.3 already
 
+PATCHES=( "${FILESDIR}/${P}-py37.patch" )
+
 S="${WORKDIR}/${MY_P}"
 
 src_prepare() {
```
Comment (SoapGentoo) [inline media-sound/beets/beets-1.4.9-r1.ebuild]: ditto

**Comment by lovesegfault** [comment]
ACK, LGTM.

**Pull request closed without merging**